sdk/metric/config.go

```diff
@@ -76,10 +79,16 @@ func unifyShutdown(funcs []func(context.Context) error) func(context.Context) er
 
 // newConfig returns a config configured with options.
 func newConfig(options []Option) config {
-	conf := config{res: resource.Default()}
+	conf := config{
+		res:            resource.Default(),
+		exemplarFilter: exemplar.TraceBasedFilter,
+	}
 	for _, o := range options {
 		conf = o.apply(conf)
 	}
+	for _, o := range optionsFromEnv() {
```

Comment:
These are applied prior to options passed directly for the trace SDK:

https://github.com/open-telemetry/opentelemetry-go/blob/d37e1ef81c2f4482c7fb2ae686f3be7ffd4bca10/sdk/trace/provider.go#L99-L103

We should unify on a precedence.

Reply:
SIG meeting note:

- Unify on the existing precedence: passed options take precedence over env var.